tests/test_mission.py

```diff
@@ -872,3 +872,15 @@ def preset_preserved(group: dcs.unitgroup.StaticGroup,
 
         static_groups = m2.coalition["blue"].countries["USA"].static_group
         self.assertTrue(any(preset_preserved(g, 3) for g in static_groups))
+
+    def test_smoke_required_modules(self) -> None:
+        mizname = "missions/Mission_with_required_modules.miz"
+        m = dcs.mission.Mission()
+        m.load_file(mizname)
+        self.assertTrue(len(m.requiredModules.keys()) == 1)
```

Comment:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertListEqual

Better to assert that the contents are actually preserved :)

Reply:
It's a dictionary, not a list, hence assertEqual

Reply:
The exact type wasn't my point though. Assert the contents, not the size. You've still got `assertTrue(something about the size)` where instead of `assertEqual(expected, the contents)`